bin/main.c

```diff
@@ -143,6 +143,49 @@ clt_psk_cb(void *m, char **username, uint8_t **key)
   return keydup(o, key);
 }
 
+static int
+cert_cb(void *m,
+        char **cert_uri,
+        char **key_uri,
+        char **pin)
```

Comment:
Please follow argument wrapping style in the code. This goes for all functions below.